src/pages/Signup/Signup.scss

```
@@ -0,0 +1,234 @@
+@import '../../styles/variables';
+@import '../../styles/mixin';
+@import url('https://fonts.googleapis.com/css2?family=Noto+Sans:wght@400;600;700&display=swap');
+
+.signup {
+  display: grid;
+  box-sizing: border-box;
+  grid-template-columns: 2fr 3fr;
+  font-family: 'Noto Sans', sans-serif;
+
+  .section-left {
+    display: flex;
+    justify-content: center;
+    padding: 32px 120px;
+
+    .column-top {
+      display: flex;
+      align-items: center;
+
+      .arrow-left {
+        position: absolute;
+        left: 32px;
+        width: 22px;
+        height: 22px;
+      }
+
+      .wekea-logo {
+        width: 100px;
+        height: 40px;
+        background-image: url('../../assets/wekea-logo.png');
+        background-size: cover;
+        cursor: pointer;
+      }
+    }
+
+    .title {
+      margin: 0;
+      margin: 24px 0;
+      font-family: 'Noto Sans ', sans-serif;
+      font-size: 32px;
+      font-weight: 700;
+
+      .ikea-blue {
+        color: $ikea-blue;
+      }
+    }
+
+    .login {
+      display: flex;
+      align-items: center;
+
+      font-size: 14px;
+      font-weight: 400;
+
+      .link {
+        color: #484848;
+      }
+    }
+
+    .images-container {
+      display: grid;
+      grid-template-columns: 1fr 1fr;
+      margin: 36px 0;
+      column-gap: 5px;
+
+      .images-column {
+        .image-reg {
+          width: 100%;
+          height: 200px;
+          margin: 5px 0;
+          border-radius: 5px;
+          background-image: url('https://images.pexels.com/photos/1128318/pexels-photo-1128318.jpeg?cs=srgb&dl=pexels-agung-pandit-wiguna-1128318.jpg&fm=jpg');
+          background-repeat: no-repeat;
+          background-position: center center;
+          background-size: cover;
+        }
+        .image-short {
+          width: 100%;
+          height: 150px;
+          margin: 5px 0;
+          border-radius: 5px;
+          background-image: url('https://images.pexels.com/photos/1128318/pexels-photo-1128318.jpeg?cs=srgb&dl=pexels-agung-pandit-wiguna-1128318.jpg&fm=jpg');
+          background-repeat: no-repeat;
+          background-position: center center;
+          background-size: cover;
+        }
+      }
```

```suggestion
.images-column {
        .image {
          width: 100%;
          margin: 5px 0;
          border-radius: 5px;
          background-image: url('https://images.pexels.com/photos/1128318/pexels-photo-1128318.jpeg?cs=srgb&dl=pexels-agung-pandit-wiguna-1128318.jpg&fm=jpg');
          background-repeat: no-repeat;
          background-position: center center;
          background-size: cover;
        }

        .reg {
          height: 200px;
        }

        .short {
          height: 150px;
        }
      }
```

스타일 부분도 위의 리뷰를 반영하면 이렇게 수정할 수 있겠네요!